app/i18n/it/index.php

```diff
@@ -27,6 +27,11 @@
 	),
 	'feed' => array(
 		'empty' => 'Non ci sono articoli da mostrare.',
+		'received' => array(
+			'before_yesterday' => 'Ricevuto prima di ieri',
```

Comment:
```suggestion
			'before_yesterday' => 'Meno recenti',
```
@Alkarex Thank you for the answer to my question then it is more correct to use the old phrase because in Italian it is not very common to use this `Ricevuto prima di ieri`

I used this translation `Ricevuto prima di ieri` first because I was not sure of the context now that I know I can make a more “correct” translation

Reply:
I suggest we stay with the more literal translation for now. We may revise those headers later (I think we have a ticket somewhere)